src/ui/views/ImportMnemonics.tsx

```diff
@@ -37,35 +167,16 @@ const ImportMnemonic = () => {
     },
   });
 
-  const handleLoadCache = async () => {
-    const cache = await wallet.getPageStateCache();
-    if (cache && cache.path === history.location.pathname) {
-      form.setFieldsValue(cache.states);
-    }
-  };
-
-  const handleValuesChange = (states) => {
-    wallet.setPageStateCache({
-      path: history.location.pathname,
-      params: {},
-      states,
-    });
-  };
-
-  const handleClickBack = () => {
-    if (history.length > 1) {
-      history.goBack();
-    } else {
-      history.replace('/');
-    }
-  };
-
-  const init = async () => {
-    if (await wallet.hasPageStateCache()) handleLoadCache();
-  };
-
   useEffect(() => {
-    init();
+    (async () => {
+      if (await wallet.hasPageStateCache()) {
+        const cache = await wallet.getPageStateCache();
+        if (cache && cache.path === history.location.pathname) {
+          setMnemonics(form.getFieldValue('mnemonics'));
+          form.setFieldsValue(cache.states);
```

Comment:
This doesn't work, it's wired

Reply:
it didn't work but not because of here, it's a UI bug in JSX, I have fixed it.